cosmos-tidoop-api/README.md

```diff
@@ -0,0 +1,209 @@
+
+[Top](#top)
+
+##<a name="section5"></a>Usage
+Please refer to this [Apiary](http://docs.tidoopmrlibapi.apiary.io/#) documentation.
+
+Please observe the MapReduce jobs usually take some time to return a result. This is why the cosmos-tidoop-api operations run the job, but do not return any result, except a `200 OK` (if the job could be successfully run) and a job identifier. In order to get the result of the opearion (or its progress), such a job identifier must be used for querying the API later.
+
+[Top](#top)
+
+##<a name="section6"></a>Administration
+Two are the sources of data, the logs and the database, useful for an administrator of cosmos-tidoop-api.
+
+[Top](#top)
+
+###<a name="section6.1"></a>Logging traces
+Logging traces, typically saved under `/var/log/cosmos/cosmos-tidoop-lib`, are the main source of information regarding the GUI performance. These traces are written in JSON format, having the following fields: level, message and timestamp. For instance:
+
+    {"level":"info","message":"Connected to http://130.206.81.225:3306/cosmos_gui","timestamp":"2015-07-31T08:44:04.624Z"}
+
+Logging levels follow this hierarchy:
+
+    debug < info < warn < error < fatal
+    
+Within the log it is expected to find many `info` messages, and a few of `warn` or `error` types. Of special interest are the errors:
+
+* ***There was some error when connecting to MySQL database. The server will not be run***: This message may appear when starting the API. Most probably the MySQL endpoint is not correct, the MySQL user is not allowed to remotely connect, of there is some network error like a port filtering.
+* ***Some error occurred during the starting of the Hapi server***: This message may appear when starting the API. Most probably the configured host IP address/FQDN does not belongs to the physical machine the service is running, or the configured port is already used.
+* ***The new job could not be added to the database***: This message may appear once a MapReduce job has been launched and it is wanted to be tracked throught the database. Most probably the connection with database has been lost, or the MySQL service is unavailable.
+* ***The MR job could not be run***: This message may appear when a new MapReduce job from tidoop-mr-lib is wanted to be run. Please check the logs generated by the library since it is not a problem related to the API.
+* ***Could not get job information for the given job_id***: This message may appear when the status/progress of a job is requested. Most probably the connection with database has been lost, or the MySQL service is unavailable.
+
+[Top](#top)
+
+###<a name="section6.2"></a>Database
+Information regarding launched jobs can be found in a MySQL table named `tidoop_job` within a database named `cosmos` in the MySQL deployment you did when installing cosmos-tidoop-api. Such a table contains the jod ID, the job type, the stating and ending timestamp the progress in terms of map and reduce percentage.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/a46e1731daedd1c5d9c75e1ca08e90be37aff51a